app/pb_kits/playbook/pb_popover/_popover.jsx

```diff
@@ -73,13 +79,13 @@ const Popover = ({
 
 export default class PbReactPopover extends React.Component<PbPopoverProps> {
   static defaultProps = {
-    modifiers: {},
+    modifiers: [],
     offset: false,
     placement: 'left',
     portal: 'body',
     show: false,
     shouldClosePopover: noop,
-    usePortal: false,
+    usePortal: true,
```

Comment:
We should use this by default everywhere for this kit. This means that now the popover content will be sourced in as the last element inside `<body>` ensuring that no z-index issues occur.